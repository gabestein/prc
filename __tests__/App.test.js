import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';

import App from '../pages/index';

describe('With Enzyme', () => {
	it('Unauthed app shows "Login page"', () => {
		const app = shallow(<App />);

		expect(app.contains(<p>Please log in.</p>)).toBe(true);
	});
});

describe('With Snapshot Testing', () => {
	it('App shows "Hello, Sunshine!"', () => {
		const component = renderer.create(<App />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
