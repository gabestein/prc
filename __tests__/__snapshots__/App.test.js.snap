// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Hello, Sunshine!" 1`] = `
<p>
  Please log in.
</p>
`;
